test/test_hpyerr.py

```diff
@@ -1,4 +1,5 @@
-from .support import HPyTest, SUPPORTS_SYS_EXECUTABLE, trampoline
+import pytest
```

Comment:
@mattip same question as for test_cpy_compat, do these need to run as apptests in pypy? If so I will pull the skip into the body of the methods like in test_object.py etc